java/ql/src/Metrics/Summaries/LinesOfCode.ql

```diff
@@ -1,8 +1,8 @@
 /**
  * @id java/summary/lines-of-code
- * @name Total lines of Java code in the database
- * @description The total number of lines of code across all Java files. This is a useful metric of the size of a database.
- *              For all Java files that were seen during the build, this query counts the lines of code, excluding whitespace
+ * @name Total lines of Java+Kotlin code in the database
```

Comment:
Hmm, this aligns with the display name of the extractor.  But perhaps it's less clear.  Happy with either one.

```suggestion
 * @name Total lines of Java/Kotlin code in the database
```